pkg/controller/controller.go

```diff
@@ -94,8 +94,10 @@ type Controller struct {
 	kubeclientset kubernetes.Interface
 	// customclientset is a clientset for our own API group
 	customclientset            clientset.Interface
-	deploymentsLister          appslisters.DeploymentLister
+	deploymentLister           appslisters.DeploymentLister
 	deploymentsSynced          cache.InformerSynced
+	replicaSetLister           appslisters.ReplicaSetLister
+	replicaSetsSynced          cache.InformerSynced
```

Comment:
`deploymentLister`(singular) and `replicaSetsLister` (plural). Is it intentional?